documentation-site/examples/badge/offsets.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import * as React from 'react';
+import {Avatar} from 'baseui/avatar';
+import {Badge, SHAPE, COLOR, PLACEMENT} from 'baseui/badge';
+
+const styles = {
+  height: '140px',
+  width: '140px',
+  border: '1px solid grey',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+};
+
+export default function Example() {
+  return (
+    <React.Fragment>
+      <Badge
+        content="Promotion"
+        shape={SHAPE.pill}
+        color={COLOR.positive}
+        placement={PLACEMENT.bottom}
+        verticalOffset="8px"
+      >
+        <div style={styles}>
```

Comment:
Does this work without the wrapper?

Reply:
I'm not sure I understand the question. We can certainly remove the div with the styles that wraps Avatar -- Badge will still work as expected. I included it because it a situation with an image or icon inlayed inside a tile or box seems like a situation where someone might want to manually position the badge.